k8s/conf/jaeger-node.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
```

Comment:
Allow local tests to use JAEGER to produce traced :-)

Reply:
Oh aren't we killing these conf files? Shoudl we this be integrated in helm?

Reply:
Not yet, this is for developers, probable just need to remove it and have locally.